app/src/main/java/org/mifos/selfserviceapp/ui/fragments/HomeScreenFragment.java

```diff
@@ -0,0 +1,172 @@
+package org.mifos.selfserviceapp.ui.fragments;
+
+import android.os.Bundle;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.models.accounts.loan.LoanAccount;
+import org.mifos.selfserviceapp.models.accounts.savings.SavingAccount;
+import org.mifos.selfserviceapp.models.accounts.savings.SavingsWithAssociations;
+import org.mifos.selfserviceapp.models.accounts.share.ShareAccount;
+import org.mifos.selfserviceapp.models.client.Client;
+import org.mifos.selfserviceapp.presenters.HomePresenter;
+import org.mifos.selfserviceapp.ui.activities.base.BaseActivity;
+import org.mifos.selfserviceapp.ui.adapters.ViewPagerAdapter;
+import org.mifos.selfserviceapp.ui.fragments.base.BaseFragment;
+import org.mifos.selfserviceapp.ui.views.HomeView;
+import org.mifos.selfserviceapp.utils.Constants;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by michaelsosnick on 1/3/17.
+ */
+
+public class HomeScreenFragment extends BaseFragment implements HomeView {
```

Comment:
This is exact copy of the [ClientAccountsFragment](https://github.com/openMF/self-service-app/blob/development/app/src/main/java/org/mifos/selfserviceapp/ui/fragments/ClientAccountsFragment.java).

Remove this class and Use the old one that is making more sense and everything is good.